domains/anhvt.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "anhvt3010"
+    },
+    "record": {
+        "CNAME": "https://build-cv.vercel.app"
```

```suggestion
        "CNAME": "build-cv.vercel.app"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).